pages/common/vifm.md

```diff
@@ -0,0 +1,17 @@
+# vifm
+
+> Vifm (VI File Manager) is a command line file manager.
+> For icons, themes, file previews and more, check out <https://github.com/vifm/vifm#other-resources>.
+> Check out the cheatsheet at <https://vifm.info/cheatsheets.shtml>.
+
+- Open current directory:
```

Comment:
```suggestion
- Open the current directory:
```

Reply:
@AfzGit Hi, this suggestion wasn't applied.

Reply:
sorry about that.